src/components/icon-button/icon-button.tsx

```diff
@@ -9,7 +9,7 @@ import { Component, Element, h, Prop } from '@stencil/core';
  */
 @Component({
     tag: 'limel-icon-button',
-    shadow: true,
+    shadow: false,
```

Comment:
Why do we have to make it "un-shadowed"? 🤔

Reply:
We forgot to get it back!. The PR is still in draft and not ready for review I think

Reply:
no no @Befkadu1 , it was intentionally, @Kiarokh suggested to play with shadow-root here.  But this hack only works with the new onClick event, which you and I were working on.  I and Kia wanted to have Adrian’s input earlier to not dig deeper if it's not the right way.